src/_data/team/stephen-mclaughlin.json

```diff
@@ -1,7 +1,7 @@
 {
     "order": 4,
     "name": "Steve McLaughlin",
-    "title": "Node-RED OSS Developer",
+    "title": "Senior Software Developer",
```

Comment:
It's a bit surprising to find this line in this PR

Reply:
Yeah. It was opportunistic. Nick told me about the role name alignment & I happened upon this during the work and slipped it in. Naughty boy.